kong/conf_loader.lua

```diff
@@ -587,6 +608,25 @@ local function load(path, custom_conf)
     end
   end
 
+  local header_tokens_clone = tablex.deepcopy(header_tokens)
+  -- load headers configuration
+  for _, token in ipairs(conf.headers) do
+    if token ~= "off" then
+      header_tokens_clone[token] = true
+    end
```

Comment:
I'm not sure how this behaviour is different from yesterday's version of this patch? It still doesn't seem like it follows the same approach as other array-like directives that accept `off`?

Reply:
Oh, I see. If the first element is `off` then turn it off else don't. My bad.